### DISCUSSION TITLE
Do Jobs work on iOS Simulator?

### DISCUSSION THREAD

**Discussion by needledrag** [shinyorg/shiny#1260] [Q&A] [answered]
Hi all,

Just testing jobs with beta version, Android works a treat but iOS on simulator won't start up, get's a Not Supported message in the Emulator console when starting up.  Checked the Shiny code for jobs and there's

`#if IOS

        else if (ObjCRuntime.Runtime.Arch == ObjCRuntime.Arch.SIMULATOR)

            result = AccessState.NotSupported;

#endif
`

Just double checking I'm on the right path here and I should test on a real device instead.

Any help much appreciated
Shaun

**Comment by aritchie** [comment, marked as answer]
The code you have linked already has your answers
